src/NServiceBus.Testing/Saga.cs

```diff
@@ -304,20 +305,36 @@ public Saga<T> When<TMessage>(Func<T, Func<TMessage, IMessageHandlerContext, Tas
         /// Invokes the saga timeout passing in the last timeout state it sent
```

Comment:
How about getting the word simulate into the test:

"Expires requested timeouts for the saga by simulating that time has passed"

and then change the doco for the `after` parameter accordingly?